<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sonic Scholar</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../assets/icon.qrc">
    <normaloff>:/logo/logo/logo_noback.png</normaloff>:/logo/logo/logo_noback.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #ffffff;
}

#left_side QPushButton {
  background-color: transparent;
  border: none;
}

#left_side QPushButton:hover {
  background-color: #F5F5F5;
  border-radius: 5px;
}

#main_loginPage #login_page_widget{
	background-color: transparent;
}

#main_loginPage QLineEdit {
	border-radius: 6px;
	border: 2px solid rgb(67, 185, 181);
}

#main_loginPage QPushButton {
	border-radius: 6px;
	border: 2px solid rgb(67, 185, 181);
	background-color:  rgb(67, 185, 181);
}

#main_loginPage QPushButton:hover {
	border-radius: 6px;
	border: 2px solid rgb(54, 150, 145);
	background-color: rgb(54, 150, 145);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="main_loginPage">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QFrame" name="frame">
          <property name="maximumSize">
           <size>
            <width>468</width>
            <height>415</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QStackedWidget" name="loginStackedWidget">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="login_page_widget">
              <widget class="QLabel" name="logoLabel">
               <property name="geometry">
                <rect>
                 <x>120</x>
                 <y>40</y>
                 <width>201</width>
                 <height>111</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="../assets/icon.qrc">:/logo/logo/logo.jpeg</pixmap>
               </property>
              </widget>
              <widget class="QLineEdit" name="input_email">
               <property name="geometry">
                <rect>
                 <x>110</x>
                 <y>200</y>
                 <width>221</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="placeholderText">
                <string>Email</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="input_password">
               <property name="geometry">
                <rect>
                 <x>110</x>
                 <y>250</y>
                 <width>221</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Password</string>
               </property>
              </widget>
              <widget class="QPushButton" name="loginBtn">
               <property name="geometry">
                <rect>
                 <x>110</x>
                 <y>310</y>
                 <width>221</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Login</string>
               </property>
              </widget>
              <widget class="QPushButton" name="registerBtn">
               <property name="geometry">
                <rect>
                 <x>160</x>
                 <y>350</y>
                 <width>121</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: transparent;
border: none;</string>
               </property>
               <property name="text">
                <string>Create an account?</string>
               </property>
              </widget>
              <widget class="QLabel" name="errorLabel">
               <property name="geometry">
                <rect>
                 <x>110</x>
                 <y>180</y>
                 <width>221</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">color: red;</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </widget>
             <widget class="QWidget" name="register_page_widget">
              <widget class="QPushButton" name="createBtn">
               <property name="geometry">
                <rect>
                 <x>110</x>
                 <y>330</y>
                 <width>221</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Create Account</string>
               </property>
              </widget>
              <widget class="QPushButton" name="alreadyloginBtn">
               <property name="geometry">
                <rect>
                 <x>110</x>
                 <y>370</y>
                 <width>221</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: transparent;
border: none;</string>
               </property>
               <property name="text">
                <string>Already have an account? login</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="input_email_register">
               <property name="geometry">
                <rect>
                 <x>110</x>
                 <y>150</y>
                 <width>221</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="placeholderText">
                <string>Email</string>
               </property>
              </widget>
              <widget class="QLabel" name="logoLabel_2">
               <property name="geometry">
                <rect>
                 <x>120</x>
                 <y>0</y>
                 <width>201</width>
                 <height>111</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="../assets/icon.qrc">:/logo/logo/logo.jpeg</pixmap>
               </property>
              </widget>
              <widget class="QLineEdit" name="input_password_register">
               <property name="geometry">
                <rect>
                 <x>110</x>
                 <y>230</y>
                 <width>221</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Enter password</string>
               </property>
              </widget>
              <widget class="QLabel" name="errorLabel_register">
               <property name="geometry">
                <rect>
                 <x>110</x>
                 <y>128</y>
                 <width>221</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLineEdit" name="conf_password_register">
               <property name="geometry">
                <rect>
                 <x>110</x>
                 <y>270</y>
                 <width>221</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Confirm password</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="input_username_register">
               <property name="geometry">
                <rect>
                 <x>110</x>
                 <y>190</y>
                 <width>221</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="placeholderText">
                <string>Username</string>
               </property>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="main_mainPage">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="left_side">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../assets/icon.qrc">:/logo/logo/logo_2.jpeg</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="main">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="spacing">
               <number>50</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <spacer name="verticalSpacer_up">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="homeBtn">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../assets/icon.qrc">
                  <normaloff>:/icons/icons/home.svg</normaloff>:/icons/icons/home.svg</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="toolsBtn">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../assets/icon.qrc">
                  <normaloff>:/icons/icons/grid.svg</normaloff>:/icons/icons/grid.svg</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="settingsBtn">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../assets/icon.qrc">
                  <normaloff>:/icons/icons/settings.svg</normaloff>:/icons/icons/settings.svg</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="helpBtn">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../assets/icon.qrc">
                  <normaloff>:/icons/icons/help-circle.svg</normaloff>:/icons/icons/help-circle.svg</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_down">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget_2">
          <widget class="QWidget" name="page"/>
          <widget class="QWidget" name="page_2"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../assets/icon.qrc"/>
 </resources>
 <connections/>
</ui>
